webhook-update.js

```diff
@@ -0,0 +1,53 @@
+
+/* update existing hooks */
+const doAsync = require("doasync");
+var GH = require("./gh");
+var config = require("./config.json");
+var hookURL = config.hookURL;
+
+
+if (require.main === module) {
+    var Store = require("./store");
+    config.logToConsole = false;
+    const store = new Store(config);
+    store.repos(async (err, data) => {
+        for (repo of data) {
+            const owner = repo.owner;
+            const shortname = repo.name;
+            const { token } = await doAsync(store).getToken(owner);        
+            const gh = new GH({ accessToken: token });
+            try {
+                const { data: hooks } = await gh.octo.request("GET /repos/:owner/:name/hooks",
+                {
+                    owner: owner,
+                    name: shortname
+                });
+
+                const hook = (hooks || hooks.length) ? hooks.find(function(h) { return h && h.config && h.config.url === hookURL; }) : null;
```

Comment:
This may not need to be taken into account explicitly (since it will show up as part of the error path), but it may be that some repos are still using the old url for the webhook (`https://labs.w3.org/hatchery/repo-manager/api/hook`)

Reply:
Indeed, I forgot that possibility. I addressed it in https://github.com/w3c/ash-nazg/pull/224/commits/2d70617d87ca1aa4f0b68f01bbab15039ddcbd2a.
Note that I think it's a good opportunity to update these hooks for the new URL.